spot_wrapper/calibration/calibration_util.py

```diff
@@ -668,6 +693,8 @@ def stereo_calibration_charuco(
             reference_img, charuco_board, aruco_dict
         )
 
+        if not no_poses:
```

Comment:
I'm a little confused by this. 
Case 1) `no_poses` is true, so `poses = list(range(len(origin_images))`. Then `filtered_poses` doesn't exist and it doesn't run
Case 2) `no_poses` is false, so the `poses` library exists and each `pose` in `poses` is added to the filtered_poses. So why not just set `filtered_poses = poses`? 

I might be missing something but I feel like we can get rid of `filtered_poses` or something?

Reply:
if no_poses is true, then poses is just filled up with dummy values so that 

when I do  zip(origin_images, reference_images, poses):

it is possible to iterate over origin images and reference images even if poses is not supplied (otherwise zip would error out on the nonetype).

As for filtered poses being the same as poses, I forgot to add the check that 

if not no_poses and origin_charuco_corners is not None:
    filtered_poses.append(pose)

forgot the second half of that (only has if not no_poses) - only want to use a pose if corners detected on location. Good catch! Reordering this to be cleaner!